tests/test_middleware_interface.py

```diff
@@ -110,9 +110,11 @@ def test_init(self):
         #   collections, etc?
         # * On init, is the local butler repo purely in memory?
 
+        # TODO: this no longer works because we cannot register an instrument
+        # and also import an export that contains that instrument.
         # Check that the butler instance is properly configured.
-        instruments = list(self.butler.registry.queryDimensionRecords("instrument"))
-        self.assertEqual(instname, instruments[0].name)
+        # instruments = list(self.butler.registry.queryDimensionRecords("instrument"))
+        # self.assertEqual(instname, instruments[0].name)
```

Comment:
I don't understand what is accomplished by commenting out these assertions. The `queryDimensionRecords` call should not attempt to modify the registry...

Reply:
We can't make this check after `init` because nothing exists after `init` now, because we can't register the instrument before the `import`.